tljh/user.py

```diff
@@ -3,11 +3,20 @@
 
 Supports minimal user & group management
 """
-import pwd
 import grp
+import pwd
 import subprocess
 from os.path import expanduser
 
+import pluggy
+
+from tljh import hooks
+
+# Set up plugin infrastructure
```

Comment:
We already instantiate a PluginManager object here: https://github.com/jupyterhub/the-littlest-jupyterhub/blob/master/tljh/jupyterhub_config.py#L72. users.py gets imported from the same place. I'd like us to have only one PluginManager object per process. Can you put it in utils.py maybe?

Reply:
Good call!